packages/react-mutation-mapper/src/component/column/Annotation.tsx

```diff
@@ -231,6 +231,12 @@ export function getAnnotationData(
                 oncoKbIndicator,
                 oncoKbAvailableDataTypes,
             };
+        } else if (oncoKbData && oncoKbData.isPending) {
```

Comment:
We need to add this because we don't catch pending status for `oncoKbData` correctly. If `oncoKbGeneExist` is false then it directly goes to 
```
else {
    value = {
        ...value,
        oncoKbStatus: 'complete',
        oncoKbIndicator: undefined,
    };
}
```
But at this moment `oncoKbData` could still be pending